README.md

```diff
@@ -283,6 +283,18 @@ principles of "convention over configuration" and separation of concerns. We
 also learned how to implement AMS with a single model. In the next lesson, we'll
 look at using AMS to serialize associations.
 
+## Check For Understanding
+
+Before you move on, make sure you can answer the following questions:
+
+1. How do we implement a serializer in our app using Active Model Serializer?
+2. What additional steps are required to return a custom method in our
+   serializer?
+3. What additional argument do we use in our `render` statement if we want to
+   specify which serializer to use?
+4. What do we mean when we say Active Model Serializer uses a convention-based
+   approach?
```

Comment:
Hm, I'm struggling with these a bit - asking students to write out all the steps to implement X feature is a bit much for the purposes of these questions. Suggestion:

1. What do we mean when we say `ActiveModel::Serializer` uses a convention-based
   approach?
2. What are some ways to break convention when using `ActiveModel::Serializer`?